.github/workflows/rds_exporter.yml

```diff
@@ -0,0 +1,59 @@
+#================================ Workflow =========================================#
+#                                                                                   #
+# This flow used to build docker image for rds_exporter and push to Docker Hub      #
+#                                                                                   #
+# Author: Dung Pham                                                                 #
+#===================================================================================#
+
+name: = BUILD IMAGE rds_exporter =
```

Comment:
we already have https://github.com/percona/rds_exporter/blob/main/.github/workflows/release.yml#L32

So this one can't be accepted, especially that it pushes to your repo.

Reply:
Oh sorry. My origin PR doesn't include it. let me check and exclude it. Thanks.